Makefile.core.mk

```diff
@@ -367,6 +367,8 @@ gen-kustomize:
 		-f manifests/charts/global.yaml > manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml
 	helm3 template istiod-remote --namespace istio-system manifests/charts/istiod-remote \
 		-f manifests/charts/global.yaml > manifests/charts/istiod-remote/files/gen-istiod-remote.yaml
+	helm3 template operator --namespace istio-system manifests/charts/istio-operator \
+		-f manifests/charts/global.yaml --set hub=gcr.io/istio-testing --set tag=${VERSION} > manifests/charts/istio-operator/files/gen-operator.yaml
```

Comment:
Why do you need to set hub and tag here?

Also, should you update the hub and tag values here to be empty `""`? https://github.com/istio/istio/blob/abedace6d38d699e32788daf9d8523ef66532224/manifests/charts/istio-operator/values.yaml#L1-L2

Reply:
the hub in istio-operator/values.yaml is (or should be) changed by the build system to istio.io. I believe that is the case already. the --set hub=gcr.io/istio-testing should be removed in this PR. With those two changes, everything should be grand.

cc/ @howardjohn thoughts on this approach re sedification in the release process?

Reply:
I would suggest we are consistent with the non-operator stuff. If that is still `sed` in the release process, we should do that -- or if we don't like it, we should change the non-operator stuff.

I don't have strong opinions here, just should be consistent

Reply:
Yeah, my suggestion to change those values to `""` is to be consistent with the values in the other `values.yaml` files.